src/Components/Screen.js

```diff
@@ -1,13 +1,34 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 class Screen extends React.Component {
   render() {
+    const { total, next, operation } = this.props;
+
+    const handleDisplay = () => {
+      let string = ''; let
+        assigned = false;
+
+      if (total) { string += total; assigned = true; }
+      if (operation) { string += operation; assigned = true; }
+      if (next) { string += next; assigned = true; }
+
+      if (!assigned) return '0';
+      return string;
+    };
+
     return (
       <div className="screen">
-        0
+        <p>{handleDisplay()}</p>
       </div>
     );
   }
 }
 
+Screen.propTypes = {
+  total: PropTypes.string.isRequired,
+  next: PropTypes.string.isRequired,
+  operation: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] There are about 3 console errors which I'll show below 👇 as a result of your PropTypes.
![image](https://user-images.githubusercontent.com/46839548/212124985-646c8744-cf96-4187-a2b9-d4d436ec13a9.png)
1. the `total` prop should be a number and not a string, so please specify `PropTypes.number.isRequired`
2. when the page first loads and you have not begun any arithmetic operation, the value of `next` prop is `null` so specifying in your PropTypes that `next` is required will be problematic. Please specify `PropTypes.string` only.
3.  same as number 2 above. Please specify `operation` prop to be `PropTypes.string` only

Having done that, you will notice that you need a `defaultProps` declaration. Kindly do this 👇 
```javascript
Screen.defaultProps = {
  next: null,
  operation: null,
};
```
Please visit this [link](https://www.geeksforgeeks.org/reactjs-defaultprops/) if you need to research more on React defaultProps 😎